defra_ruby_aws.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |spec|
   spec.test_files = Dir["spec/**/*"]
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.17"
+  spec.add_development_dependency "bundler"
```

Comment:
I don't normally bother declaring bundler as a developer dependency as I feel its a given you're gonna need it.

Reply:
No, not really. It was a copy-paste. Let's remove it :) 
```suggestion
```